### PR TITLE
Added the acid2 example and some fixes to render it quite nice

### PR DESCRIPTION
**Pull request by cornedor**
Firefox (left) and Chrome (right).
Acid2 renderd in Firefox (left) and Chrome (right). 

The eye's are done very tricky in acid2 (a object element in a object element in a object element)

I've added support for background-attachment: fixed and a fix for Chrome using 0 as default z-index.

### CODE REVIEW

File: src/nodeparser.js [hunk 1]
```diff
@@ -764,6 +764,10 @@ function renderableNode(node) {
 function isPositionedForStacking(container) {
     var position = container.css("position");
     var zIndex = (["absolute", "relative", "fixed"].indexOf(position) !== -1) ? container.css("zIndex") : "auto";
+    // In chrome fixed has a default zIndex of 0 instead of auto
+    if(parseFloat(zIndex, 10) === 0) {
```
Comment (niklasvh) [inline src/nodeparser.js]: `parseFloat` doesn't have a second argument like `parseInt` has, and I believe `parseInt` would be more suitable here anyway.

File: src/nodecontainer.js [hunk 2]
```diff
@@ -164,6 +164,7 @@ NodeContainer.prototype.parseBackgroundSize = function(bounds, image, index) {
 
 NodeContainer.prototype.parseBackgroundPosition = function(bounds, image, index, backgroundSize) {
     var position = this.cssList('backgroundPosition', index);
+    var attachment = this.cssList('backgroundAttachment', index);
```
Comment (niklasvh) [inline src/nodecontainer.js]: Could you add some test for `background-attachment` as well, somewhere under https://github.com/niklasvh/html2canvas/tree/master/tests/cases/background

File: examples/acid2.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
```
Comment (niklasvh) [inline examples/acid2.html]: The acid2 test got added last week already, under https://github.com/niklasvh/html2canvas/blob/master/tests/cases/acid2.html so I guess this one isn't necessary.

**Comment by niklasvh** [comment]
Thanks for the pull request! I've added a few inline comments.

File: src/nodeparser.js (hunk 1 above)
Comment (cornedor) [inline src/nodeparser.js]: Meant to use that one indeed :) thanks for pointing out.

**Comment by cornedor** [comment]
I've added the test and removed the acid2 example, while adding the test I found a bug with background position and borders, fixed that too.

**Comment by NeXTs** [comment]
Any updates on this? I do really need support of background-attachment:fixed from this pull request.

**Comment by cornedor** [comment]
I could create a pull request with just the `background-attachment: fixed;` fix.

**Pull request closed without merging**